misc/scripts/install_d3d12_sdk_windows.ps1

```diff
@@ -0,0 +1,82 @@
+# DirectX Shader Compiler
+
+$dxc_filename="dxc_2023_08_14.zip"
+Write-Output "Downloading DirectX Shader Compiler $dxc_filename ..."
+Invoke-WebRequest -Uri https://github.com/microsoft/DirectXShaderCompiler/releases/download/v1.7.2308/$dxc_filename -OutFile "$env:TEMP\$dxc_filename"
```

Comment:
Can we extract all version strings as variables at the top of the script, so it's easy to update?